docs/src/routes/docs/utils.svx

```diff
@@ -0,0 +1,21 @@
+---
+name: Utility functions
+---
+
+# Utility functions
+
+A collection of helper functions that can be imported by name from `attractions/utils`. For example: `import { classes } from 'attractions/utils';`
+
+
+| Name | Signature | Description |
+| ---- | --------- | ----------- |
+| **`callOnSight`** | `(node, { callback, args }) => { destroy }` | An [action](https://svelte.dev/tutorial/actions) that is intended to be used with the `use` directive. Calls the provided `callback` with the given `args` (spreaded) once the component appears on the screen. |
```

Comment:
Yeah, that's what I was fearing the most. This stuff is very hard to read, but I suppose it will do for now

Reply:
I think only functions intended for use with directives (such as `use:`) are the ones difficult to read. I don't really have a better idea how to document them

Reply:
No, I meant that the description column in the table is squeezed

Reply:
We should remove the table in the future